pkg/applicationserver/io/web/templates.go

```diff
@@ -64,24 +58,55 @@ func (c TemplatesConfig) NewTemplateStore() (*TemplateStore, error) {
 		var err error
 		fetcher, err = fetch.FromHTTP(c.URL, true)
 		if err != nil {
-			return nil, err
+			return &noopTemplateStore{}, err
 		}
 	default:
-		return nil, nil
+		return &noopTemplateStore{}, nil
 	}
 	baseURL, err := url.Parse(c.LogoBaseURL)
 	if err != nil {
-		return nil, err
+		return &noopTemplateStore{}, err
 	}
-	return &TemplateStore{
+	return &templateStore{
 		fetcher:   fetcher,
 		baseURL:   baseURL,
 		templates: make(map[string]queryResult),
 	}, nil
 }
 
+type noopTemplateStore struct {
+}
+
+var (
+	errNoStore = errors.DefineUnavailable("err_no_store", "no template store")
```

Comment:
`Unavailable` is retryable for clients. This is not retryable

My suggestion would be to:
- Return an empty list on `ListTemplates()` (and not return an error)
- Always return a `NotFound` (404) error on `GetTemplate()`